spec/requests/api/v1/moves_controller_index_spec.rb

```diff
@@ -107,29 +107,9 @@
           }
         end
 
-        it 'paginates 20 results per page as default' do
-          get_moves
-
-          expect(response_json['data'].size).to eq 20
-        end
-
-        it 'returns 1 result on the second page'  do
```

Comment:
Looks like this test (of the links) is now removed. Is there a way that could be added to the shared context, perhaps by reading the link from the returned metadata?

Reply:
talked in person but for posterity: this test is defined in the shared example but using `params` instead